doc/sphinx_source/using/botnet.rst

```diff
@@ -52,7 +52,7 @@ The following are some common terms used in this document:
   Passive share is a term used to describe the direction of sharing user-files. Passive share bots will accept userfiles from an aggressive share bot.
 
 **Address**
-  The physical address, containing the address and port of the bot. For example: lame.org:3333. You can change a bot's address with::
+  The physical address, containing the address and port of the bot. For example: lame.example.org:3333. You can change a bot's address with::
```

Comment:
Please revert this file to its original state, thanks

Reply:
Names like lame.org are no good for documentation, because they can be, and actually are valid addresses, and could also be used for [Cybersquatting](https://en.wikipedia.org/wiki/Cybersquatting), so i changed them to ones given by [rfc2606](https://datatracker.ietf.org/doc/html/rfc2606), [rfc 3849](https://datatracker.ietf.org/doc/html/rfc3849) and [rfc 5737](https://datatracker.ietf.org/doc/html/rfc5737)  keeping  as much as possible. so lame.org becomes lame.example.org.

Reply:
Yep, good thoughts, and if these were active default config settings I'd go with it, but I'd prefer to keep the documentation as is. Thanks